### DISCUSSION TITLE
Accessible Multiplayer Dice Roller

### DISCUSSION THREAD

**Discussion by DDR0** [dice-roller/rpg-dice-roller#245] [Show and tell]
Hey! Just wanted to say thanks for providing the backing service for my little accessible dice roller. We've been using it for a few years now for our game and it's been great. ^_^

Check it out at [https://ddr0.ca/🎲](https://ddr0.ca/d).

**Comment by GreenImp** [comment]
This looks really good, thanks for sharing!
